### PR TITLE
DO NOT MERGE UNTIL version 1.4.0: Readme.md

### PR DESCRIPTION
**Pull request by susanmdouglas-aws**
This PR contains the Readme.md file and supporting artifacts to publish the update on our website.

Signed-off-by: susanmdouglas <susandou@amazon.com>

### Description
This PR contains the Readme.md file and supporting artifacts to publish the update on our website.
 
### Issues Resolved
This PR contains the Readme.md file and supporting artifacts to publish the update on our website.

### Check List
- [x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by 2jungkook — APPROVED** [review]

**Review by geetanjalij — DISMISSED** [review]

File: index.markdown [hunk 1]
```diff
@@ -55,6 +55,8 @@ sidebar:
         url: "/docs/versions/babelfish-2-1-0a.html"
       - title: Release 2.1.0
         url: "/docs/versions/babelfish-2-1-0.html"
+      - title: Release 1.4.0
+        url: "/docs/versions/babelfish-1-3-1.html"
```
Comment (suprio-amzn) [inline index.markdown:59]: Is this correct?

File: _versions/babelfish-1.4.0.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+date: 2022-7-25
```
Comment (suprio-amzn) [inline _versions/babelfish-1.4.0.md:2]: Please fix the date.

**Review by suprio-amzn — CHANGES_REQUESTED** [review]

File: _versions/babelfish-1.4.0.md (hunk 2 above)
Comment (susanmdouglas-aws) [reply _versions/babelfish-1.4.0.md:2]: I'll update that after we upload the installers/release notes - it's easier/safer that way.

File: index.markdown (hunk 1 above)
Comment (susanmdouglas-aws) [reply index.markdown:59]: Now that I can fix - good catch!

File: _includes/releases/babelfish-1.4.0.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+After the extensions are updated, you can check the extension version from the PostgreSQL port:
+
+```bash
+babelfish_db[124386]# \dx
+                                List of installed extensions
+        Name        | Version |   Schema   |                   Description
+--------------------+---------+------------+-------------------------------------------------
+ babelfishpg_common | 1.4.0   | public     | Transact SQL Datatype Support
+                                 ....
+ babelfishpg_tsql   | 1.4.0   | public     | Transact SQL Compatibility
```
Comment (kuntalghosh) [inline _includes/releases/babelfish-1.4.0.md:59]: The version for both the extensions are 1.2.1
https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_1_4_STABLE/contrib/babelfishpg_common/Version.config
https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_1_4_STABLE/contrib/babelfishpg_tsql/Version.config

Comment (susanmdouglas-aws) [reply _includes/releases/babelfish-1.4.0.md]: Updated...

Comment (susanmdouglas-aws) [reply _includes/releases/babelfish-1.4.0.md]: Updated...

**Review by kuntalghosh — APPROVED** [review]

**Review by shardgupta — APPROVED** [review]

**Review by 2jungkook — DISMISSED** [review]

File: _includes/releases/babelfish-1.4.0.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
+- Babelfish Version: 1.4.0
+- PostgreSQL Server Version: 13.8
+- Download source distributions:
+  - [BABEL_1_4_0__PG_13_8.zip](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_1_4_0__PG_13_8/BABEL_1_4_0__PG_13_8.zip)
+  - [BABEL_1_4_0__PG_13_8.tar.gz](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_1_4_0__PG_13_8/BABEL_1_4_0__PG_13_8.tar.gz)
+- Babelfish Compass
+  - [Download](https://github.com/babelfish-for-postgresql/babelfish_compass/releases)
+- Date: November 14, 2022
```
[the remark this answers is not in the document]
Comment (susanmdouglas-aws) [reply _includes/releases/babelfish-1.4.0.md:8]: I'll update the date (and any links that need it) after we publish...

**Review by kuntalghosh — APPROVED** [review]

**Comment by susanmdouglas-aws** [comment]
I've pushed these changes at the command line after approval from Kuntal and Shard, so am closing this PR.

**Pull request closed without merging**